src/openrct2/object/Object.h

```diff
@@ -29,6 +29,9 @@ constexpr const ObjectEntryIndex OBJECT_ENTRY_INDEX_NULL = std::numeric_limits<O
 struct ObjectRepositoryItem;
 using ride_type_t = uint16_t;
 
+constexpr const int8_t VersionNumFields = 3;
+using Version = std::tuple<uint16_t, uint16_t, uint16_t>;
```

Comment:
Just rename this to ObjectVersion, it’s clearer and allows you to undo the case change, which we don’t like.